backend/start.js

```diff
@@ -7,7 +7,7 @@ const app = require('./app/express.js');
 /* ======= Start socket service ======= */
 require('./app/socket.js');
 
-app.listen(process.env.EXPRESS_PORT, () =>
+app.listen(process.env.EXPRESS_PORT, '0.0.0.0', () =>
```

Comment:
Did you mean to leave this in? I got the feeling from your instructions this shouldn't be pushed but I could be wrong.